napari/_qt/layer_controls/qt_points_controls.py

```diff
@@ -90,6 +90,11 @@ def __init__(self, layer):
         self.layer.text.events.visible.connect(self._on_text_visibility_change)
 
         sld = QSlider(Qt.Horizontal)
+        sld.setToolTip(
+            trans._(
+                "Change the size of currently selected points and the future ones."
```

Comment:
```suggestion
                "Change the size of the currently selected points and any added in the future."
```

Reply:
I think about `any added in` but I'm not sure if it will be clear that it is any added until the next move of the slider?

Reply:
Yeah, I struggled with the wording here too. I think it would be fine just to move the `the` in-front of `currently` and keep `future ones` instead of `any added in`.